### DISCUSSION TITLE
Github Pages

### DISCUSSION THREAD

**Discussion by HerbRSPS** [community/community#32678] [Other Feature Feedback, Questions, & Ideas]
I've setup github pages before with a custom domain (my own portfolio). Now.. i made a new repo with a new project where I want to use a custom domain for as well, unfortunately I can't find a proper guide to help me with that. Anyone that can help me setting up a 2nd custom domain name page?

**Comment by airtower-luna** [comment]
Here's the documentation on how to set a custom domain: https://docs.github.com/en/pages/configuring-a-custom-domain-for-your-github-pages-site/managing-a-custom-domain-for-your-github-pages-site

**Reply by HerbRSPS** [reply]
But i already have one setup, im looking to add a second one..

**Reply by airtower-luna** [reply]
A custom domain for Pages is per-repository. Setting a second domain up for a second (different) repository is exactly the same process as for the first one. :slightly_smiling_face:

**Reply by HerbRSPS** [reply]
hmmm interesting, alright ill give it a go! thanks :)

**Reply by HerbRSPS** [reply]
I suppose it takes a bit for the github pages to be able to work, cuz last time it took some time as well before the DNS check started working.
